nuxeo-core/nuxeo-core/src/main/resources/OSGI-INF/coldstorage-contrib.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.core.event.coldstorage.contribution">
```

Comment:
maybe just `org.nuxeo.ecm.core.coldstorage.contrib` as it's not only event related, and we often use "contrib"